Studentify/Models/Authentication/RegisterModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Studentify.Models.Authentication
+{
+    public class RegisterModel
```

Comment:
To też jest model tylko do przesyłania w body z tego co rozumiem, także tak samo jak wyżej można o tym napisać w dokumentacji i pomyśleć czy nie dać do jakiegoś katalogu z nazwą httpBodyModels czy coś takiego, żeby odróżnić od DBModels (te nazwy to pierwsze co mi przyszły do głowy, może będą lepsze)

Reply:
Done jak w przypadku LoginModel